build.js

```diff
@@ -32,16 +42,60 @@ async function rmDistDir() {
   }
 }
 
-async function main() {
-  console.info('Compiling sequelize...');
-  const [declarationFiles, filesToCompile] = await Promise.all([
+function sourcePathToDestPath(source) {
+  return path.resolve(outdir, source).replace(/\.ts$/, '.js');
```

Comment:
```suggestion
  return path.resolve(outdir, source).replace(/\.([cm])?ts$/, '.$1js');
```

to support cts/mts again :)

Reply:
obsolete